src/styles/components/forms.css

```diff
@@ -163,11 +173,11 @@
   
         &:active,
         &:focus {
-          @apply border-gray-lightest ring-0 text-dark-charcoal;
+          @apply border-gray-lightest ring-0;
         }
   
         &[disabled] {
-          @apply bg-white-antiflash;
+          @apply bg-white-ish;
```

Comment:
What means ish? Try to be more descriptive in naming colors; there are many ways to name a color https://www.eggradients.com/color-names.
There are color pickers tools that name a color; when you pick for macOS, I'm using the `colorSlurp` app.

Reply:
Actually I just took the color naming style from the existing ones. 😓  I'll look into that and change it.